addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -270,20 +253,28 @@ private void requestPlayerUpdate(int activePlayer) {
             channel = item.get("channel").getAsString();
         }
 
+        String thumbnail = "";
```

Comment:
Let me explain: No, not in this place.

Currently setting the channel states to "" is the way to go in this binding. It's done with all other readonly String channel states (e.g. artist or album) too. If we want to set `UnDefType` instead we should do it in the [updateThumbnail()](https://github.com/openhab/openhab2-addons/pull/2304/files/aa38c0624395ff58e21febf978fa1a62d69c98aa#diff-b0bd6019b6836bae01ee8b4dd1e315b7R389) (and similar) method.